### PR TITLE
Yowlj presentation

### PR DESCRIPTION
**Pull request by mankyKitty**
add prezzo

### CODE REVIEW

**Comment by mankyKitty** [comment]
Added the workshop slides, lemme know if you want more... `Symbolic` & `Concrete` seem a bit trivial for slides?

File: presentation/talk.org [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  - eat the cost for some big things
+  - write unit tests for some small things
+  #+BEGIN_SRC haskell
+    getLunch "fishy" == TunaSalad
+    getLunch "hungry spicy" == ExtraHot HalfChicken
+  #+END_SRC
+  - subcontract end users to suggest regression tests
+
+* Property-based awesome
+  - unleash the power of   $\forall$
```
Comment (endgame) [inline presentation/talk.org]: 2spaces

File: presentation/talk.org [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+** Shrinking (OMG!)
+   you thought shrinking was cool before...
+
+** Useful feedback
+   a failure will produce:
+   - the sequence of actions
+   - their respective inputs
+
+* OMG
```
Comment (endgame) [inline presentation/talk.org:57]: OMG

File: presentation/talk.org [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+   - *not the thing itself*
+
+** Command
+   - represents /transitions/
+   - *not states*
+ 
+** Given
+   [[./images/mach.png]]
+
+** Test this
```
Comment (endgame) [inline presentation/talk.org]: Swap these slides: "don't test this"/"test this"? That way transitions are in the audience's mind as you go on.

File: presentation/talk.org [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+* Examples
+  - Wordpress API (a tiny bit of it)
+  - LevelDB
+
+* Come to the workshop!
+  Learn the weirding way!
+  - https://github.com/qfpl/state-machine-testing-course/
+
+* Bonus Round
+ Use classification to understand your test-case distribution
```
Comment (endgame) [inline presentation/talk.org]: indent

File: presentation/workshop.org [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+#+TITLE: Property-based state machine testing
+#+AUTHOR: Sean Chalmers, Jack Kelly
+#+EMAIL:
+
+* Workshop
+  https://github.com/qfpl/state-machine-testing-course
+  
+* Command
+  #+BEGIN_SRC haskell
+    data Command n m (state :: (* -> *) -> *) = forall input output.
```
Comment (endgame) [inline presentation/workshop.org]: Alpha-convert `n -> g`, add comment "g is often MonadGen, m often MonadTest, MonadIO"

**Review by endgame — APPROVED** [review]
Looks pretty good

**Pull request merged**